api/openapi.yaml

```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.1
```

Comment:
should this file be checked in this repository as well as its own repo?

Reply:
This is actually an output of the generator (and no longer an input). It's fine to leave this here, and in fact it will be useful in allowing us to serve up the current spec through swagger UI in local and development environments.